www/js/plugin/logger.ts

```diff
@@ -33,6 +33,9 @@ export const logInfo = (message: string) =>
 export const logWarn = (message: string) =>
   window['Logger'].log(window['Logger'].LEVEL_WARN, message);
 
+export const logError = (message: string) =>
+  window['Logger'].log(window['Logger'].LEVEL_ERROR, message);
+
```

Comment:
I believe that as a rule, we want all caught errors to be user-facing (`displayError`/`displayErrorMsg`). That is why I had purposefully *not* implemented a `logError`.
Let me know if you've heard otherwise

Reply:
That makes sense, I'll switch these over.